tests/coverage/generate_coverage.sh

```diff
@@ -19,10 +21,12 @@ llvm-cov-7 export -instr-profile coverage.profdata -format=text "${objects[@]}"
 
 # Generate and upload combined coverage report for Codecov
 llvm-cov-7 show -instr-profile coverage.profdata "${objects[@]}" -ignore-filename-regex="(openenclave|3rdparty|/test/)" > codecov.txt
-bash <(curl -s https://codecov.io/bash) -t "${CODECOV_TOKEN}" -f codecov.txt -F unit
+bash <(curl -s https://codecov.io/bash) -t "${CODECOV_TOKEN}" -f codecov.txt -F "unit_${suffix}"
 
 for e2e in *.virtual.so; do
-    llvm-profdata-7 merge -sparse ./*_"$e2e".profraw -o "$e2e".profdata
-    llvm-cov-7 show -instr-profile "$e2e".profdata -object cchost.virtual -object "$e2e" -ignore-filename-regex="(openenclave|3rdparty|/test/)" > "$e2e".txt
-    bash <(curl -s https://codecov.io/bash) -t "${CODECOV_TOKEN}" -f "$e2e".txt -F "$(echo $"e2e" | cut -d. -f1)"
+    if [ -f ./*_"$e2e".profraw ]; then
```

Comment:
Not all end to end tests run with PBFT yet, the profraw may legitimately not exist.